src/GraphQL/ParseGraphQLSchema.js

```diff
@@ -53,6 +55,7 @@ class ParseGraphQLSchema {
       databaseController: DatabaseController,
       parseGraphQLController: ParseGraphQLController,
       log: any,
+      appId: String,
```

Comment:
This should be the primitive `string` type

Reply:
done